workspaces/report-portal/catalog-info.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: backstage.io/v1alpha1
```

Comment:
This file is intended to be consumed by Adopters and not to act as an example for the plugin itself.

Reply:
Ahh I was testing using that file 🙇‍♂️, I'll move it to examples, and restore it